pkg/client/camel/group.go

```diff
@@ -0,0 +1,25 @@
+// +build tools
+
+/*
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package camel
+
+import (
+	// We use k8s.io/code-generator tooling in external scripts. In order to keep it as a Go module, we must declare it explicitly.
+	_ "k8s.io/code-generator"
```

Comment:
Not sure if it's a good practice. Doesn't it make a build bigger unnecessarily?
Isn't just keeping `go.mod` untouched after doing `go get k8s.io/code-generator` sufficient?

Reply:
It's a very valid concern. I did not think about that side effect. Without that import, the `go mod tidy` wipes off any dependency that is not used, and we do use in `make generate`. Let me try with the approach you're suggesting, updating the script if it makes sense.